server/repository/src/db_diesel/item_direction.rs

```diff
@@ -0,0 +1,72 @@
+use super::{item_direction_row::item_direction, ItemDirectionRow, StorageConnection};
+
+use crate::diesel_macros::apply_equal_filter;
+
+use crate::{repository_error::RepositoryError, DBType, EqualFilter};
+use diesel::{dsl::IntoBoxed, prelude::*};
+
+pub type ItemDirection = ItemDirectionRow;
+
+#[derive(Clone, Default, PartialEq, Debug)]
+pub struct ItemDirectionFilter {
+    pub id: Option<EqualFilter<String>>,
```

Comment:
I think we're going to want a filter on `item_id`, could be done later though, so I'm happy if it's not in this PR.